include/oneapi/dpl/internal/async_impl/glue_async_impl.h

```diff
@@ -16,18 +16,31 @@
 #ifndef _ONEDPL_GLUE_ASYNC_IMPL_H
 #define _ONEDPL_GLUE_ASYNC_IMPL_H
 
-#include "../async_extension_defs.h"
+#if _ONEDPL_BACKEND_SYCL
+#    include "../async_extension_defs.h"
 
-#if _ONEDPL_HETERO_BACKEND
-#    include "async_impl_hetero.h"
-#endif
+#    if _ONEDPL_HETERO_BACKEND
+#        include "async_impl_hetero.h"
+#    endif
+#endif // _ONEDPL_BACKEND_SYCL
```

Comment:
Let's remove `#if` and `#endif` in the middle.
The whole idea of distinguishing SYCL backends and "generic hetero" layer seems questionable to me at this point; will we ever have a non-SYCL hetero backend?
Also from the practical standpoint `_ONEDPL_HETERO_BACKEND` is always defined to 1 if `_ONEDPL_BACKEND_SYCL` is defined, so the check in the middle is really useless.

Reply:
Fixed.